collections/ui/API.md

```diff
@@ -0,0 +1,2653 @@
+### AlertBar
+
+#### Usage
+
+**Note**: If possible, import the component from the main UI (`@dhis2/ui`) package.
+
```

Comment:
If possible, these notes should only appear in the component packages, not in the collection packages, since the import below is already correct.